apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -125,6 +131,35 @@ type Resources struct {
 	Total *corev1.ResourceRequirements `json:"total,omitempty"`
 }
 
+type SearchSpec struct {
+	// Enable tempo search feature, default to true
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Search enabled"
+	Enabled *bool `json:"enabled,omitempty"`
+	// Limit used for search requests if none is set by the caller (default: 20)
```

Comment:
in which cases none is set by the caller? E.g. jaeger-ui would not set it?

Reply:
It could be on jaeger-ui or if someone else calls directly the API.

Reply:
As a newbie, I am still confused about what this limit means. Does it limit the number of returned traces or spans or any other data types?

Reply:
It is on traces, already specified in the comment.